File: Golang/two-sum.go
package main

import "fmt"

func twoSum(nums []int, target int) []int {
	var new_array []int
	var x int
	var y int
	for i := 0; i < len(nums); i++ {
		for j := 0; j < len(nums); j++ {
			if i != j {
				if nums[i]+nums[j] == target {
					x = i
					y = j
					break
				}
			}
		}
	}

	new_array = append(new_array, x)
	new_array = append(new_array, y)

	return new_array
}

func main() {
	arr := []int{3, 3}
	x := twoSum(arr, 6)

	fmt.Println(x)
}

File: Python/twosum.py
from ast import List

class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        y = x = 0
        arr = []
        for i in range(len(nums)):
            for j in range(len(nums)):
                if i != j:
                    if nums[i] + nums[j] == target:
                        x = i
                        y = j                
        arr.append(y)
        arr.append(x)
        return arr
        